packages/table/src/interactions/dragEvents.ts

```diff
@@ -92,6 +99,9 @@ export class DragEvents {
         if (this.handler.stopPropagation) {
             event.stopPropagation();
         }
+        if (this.handler.stopImmediatePropagation) {
```

Comment:
I've run into a lot of subtle issues when messing with these. Make sure you comment your usage where you set `stopImmediatePropagation`